manager/src/main/java/com/redhat/service/bridge/manager/BridgesServiceImpl.java

```diff
@@ -67,6 +72,11 @@ public Bridge getBridge(String id, String customerId) {
 
     @Override
     public void deleteBridge(String id, String customerId) {
+        List<Processor> processors = processorService.getProcessors(id, customerId);
```

Comment:
Same comment about a `count()` method or a list of Ids instead of the whole entity

Reply:
Implemented with `getProcessorsCount` in the `ProcessorService` and `ProcessorDAO` accordingly